zsh/configs/completion.zsh

```diff
@@ -0,0 +1,6 @@
+# load our own completion functions
+fpath=(~/.zsh/completion /usr/local/share/zsh/site-functions $fpath)
+
+# completion
+autoload -U compinit
+compinit
```

Comment:
Any thought to adding this to `zsh/configs/post/completion.zsh`? Then any completions defined on functions or not-yet-loaded commands won't generate an error.